### ISSUE TITLE
[Craft 4] Email notification preview not working due to "No recipient email address found" error

### ISSUE THREAD

**Issue by joelzerner** [verbb/formie#909]
**Description**

Seeing an "An error occurred. Notification email error. No recipient email address found." whenever I go to preview the email notification even though I have a valid email address in the recipient field and it has worked well previously.

**Steps to reproduce**
1. Edit Form > Email Notifications tab > Edit Notification
A. Click Preview Tab
B. Click Preview Tab and try sending a test email (same error appears)

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: Yes

**Additional info**
- Plugin version: 2.0.0-beta.10
- Craft version: 4.0.3
- Multi-site: No

**Additional context**

![image](https://user-images.githubusercontent.com/11308275/170918451-23ed2d75-6359-441c-9e4b-45b952fdfcb9.png)

**Comment by JeanLucEsser** [comment]
Oops, didn't see that one, I filed a similar issue here: #914.
For me it's not only preview, sending does not work either.

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.0-beta.10",
  "...": "..."
}
```

Then run `composer update`.

**Comment by JeanLucEsser** [comment]
I can confirm this is fixed for me.

**Comment by engram-design** [comment]
Fixed in [2.0.0-beta.11](https://github.com/verbb/formie/releases/tag/2.0.0-beta.11)
